Public/New-TeamsSection.ps1

```diff
@@ -8,14 +8,33 @@ function New-TeamsSection {
         [string] $ActivitySubtitle ,
         [string] $ActivityImageLink,
         [string][ValidateSet('Alert', 'Cancel', 'Disable', 'Download', 'Minus', 'Check', 'Add', 'None')] $ActivityImage = 'None',
+
+        [ValidateScript({
+            if (-not ($_ | Test-Path)) {
+                throw "ActivityImagePath is inaccessible or does not exist"
+            }
+            if (-not ($_ | Test-Path -PathType Leaf) -or ($_ -notmatch "(\.jpg|\.png)")) {
+                throw "ActivityImagePath is not a file or file extension is not supported"
+            }
+            return $true
+        })]
+        [System.IO.FileInfo] $ActivityImagePath,
+
         [string] $ActivityText,
         [string] $Text,
         [System.Collections.IDictionary[]]$ActivityDetails,
         [System.Collections.IDictionary[]]$Buttons,
         [switch] $StartGroup
     )
-    if ($ActivityImage -ne 'None') {
-        $StoredImages = [IO.Path]::Combine($PSScriptRoot, '..', 'Images')
+
+    if ($ActivityImagePath) { # ActivityImagePath takes precedence over ActivityImage
+        $FilePath = [System.IO.Path]::GetDirectoryName($ActivityImagePath)
+        $FileBaseName = [System.IO.Path]::GetFileNameWithoutExtension($ActivityImagePath)
+        $FileExtension = [System.IO.Path]::GetExtension($ActivityImagePath)
+        $ActivityImageLink = Get-Image -PathToImages $FilePath -FileName $FileBaseName -FileExtension $FileExtension # -Verbose
+    }
+    elseif ($ActivityImage -ne 'None') {
+        $StoredImages = [IO.Path]::Combine($PSScriptRoot, 'Images')
```

Comment:
Apparently this line should not be changed and path is intended to be updated upon publishing. See here: https://github.com/EvotecIT/PSTeams/issues/22#issuecomment-671254832